sebs/gcp/cli.py

```
@@ -1,5 +1,24 @@
+Example:
+    Using the gcloud CLI interface:
+    
+        cli = GCloudCLI(credentials, system_config, docker_client)
+        cli.login(project_name)
+        result = cli.execute("gcloud functions list")
+        cli.shutdown()
+"""
+
 import logging
 import os
+from typing import Union
```

Ruff (0.11.9)

`typing.Union` imported but unused

Remove unused import: `typing.Union`

(F401)